app/import.go

```diff
@@ -68,7 +68,7 @@ func (a *App) BulkImport(fileReader io.Reader, dryRun bool, workers int) (*model
 		}
 
 		if line.Type != lastLineType {
-			if lastLineType != "" {
+			if lastLineType != "begin" {
```

Comment:
Not entirely sure I folllow this approach. Couldn't we just check for `if line.Type.isEmpty() { fail; }` somewhere? This feels like it isn't robust against a line with type `begin` in the data file, or am I missing something?

Reply:
Yes, that would be a possible case, really weird, but possible, I going to rethink the solution.

Reply:
Done